cluster/deploy.sh

```diff
@@ -50,7 +50,10 @@ EOF
 _kubectl apply -f ${MANIFESTS_OUT_DIR}/release/kubevirt-operator.yaml
 
 if [[ "$KUBEVIRT_PROVIDER" =~ os-* ]] || [[ "$KUBEVIRT_PROVIDER" =~ okd-* ]]; then
-    _kubectl create -f ${MANIFESTS_OUT_DIR}/testing/ocp
+
+    if [ -d "${MANIFESTS_OUT_DIR}/testing/ocp" ]; then
+        _kubectl create -f ${MANIFESTS_OUT_DIR}/testing/ocp
```

Comment:
There is also os/okd section in hack/cluster-clean.sh removing SCC, any change it is related to the change?

Reply:
mh, good catch, but it at least doesn't hurt to leave it there, in case we add SCC later on again.